docs/docs-ref-autogen/excel/excelscript.yml

```diff
@@ -332,7 +332,8 @@ functions:
     summary: >-
       Return the text encoding of the document as a PDF. If the document is
       empty, then the following error is shown: "We didn't find anything to
-      print".
+      print". Some actions made prior to using this API may not be captured in
```

Comment:
By "some actions", does this comment specifically mean actions previously done in the script or in the session?

Reply:
Thanks! I will merge this PR and consult the feature crew.